contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -424,132 +368,19 @@ library ValidationLogic {
             reserveCache.variableDebtTokenAddress
         ).getPreviousIndex(onBehalfOf);
 
-        uint40 stableRatePreviousTimestamp = IStableDebtToken(
-            reserveCache.stableDebtTokenAddress
-        ).getUserLastUpdated(onBehalfOf);
-
         require(
-            (stableRatePreviousTimestamp < uint40(block.timestamp) &&
-                interestRateMode == DataTypes.InterestRateMode.STABLE) ||
-                (variableDebtPreviousIndex <
-                    reserveCache.nextVariableBorrowIndex &&
-                    interestRateMode == DataTypes.InterestRateMode.VARIABLE),
+            (variableDebtPreviousIndex < reserveCache.nextVariableBorrowIndex &&
+                interestRateMode == DataTypes.InterestRateMode.VARIABLE),
```

Comment:
let's split this into 2 requires with the proper error, we have ProtocolError INVALID_INTEREST_RATE_MODE_SELECTED

Reply:
since we remove stable rate, I would prefer also remove `InterestRateMode` totally to make less confusing, but will introduce some api break change, but I think should be fine before we release the production @ivansolomonoff @GopherJ  what do you think?